orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/config/CloudDriverConfiguration.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.config;
+
+import static retrofit.Endpoints.newFixedEndpoint;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jakewharton.retrofit.Ok3Client;
+import com.netflix.spinnaker.config.DefaultServiceEndpoint;
+import com.netflix.spinnaker.config.okhttp3.OkHttpClientProvider;
+import com.netflix.spinnaker.kork.core.RetrySupport;
+import com.netflix.spinnaker.kork.web.selector.DefaultServiceSelector;
+import com.netflix.spinnaker.kork.web.selector.SelectableService;
+import com.netflix.spinnaker.kork.web.selector.ServiceSelector;
+import com.netflix.spinnaker.orca.api.operations.OperationsRunner;
+import com.netflix.spinnaker.orca.clouddriver.*;
+import com.netflix.spinnaker.orca.jackson.OrcaObjectMapper;
+import com.netflix.spinnaker.orca.retrofit.RetrofitConfiguration;
+import com.netflix.spinnaker.orca.retrofit.logging.RetrofitSlf4jLog;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import retrofit.RequestInterceptor;
+import retrofit.RestAdapter;
+import retrofit.converter.JacksonConverter;
+
+@Configuration
+@Import(RetrofitConfiguration.class)
+@ComponentScan({
+  "com.netflix.spinnaker.orca.clouddriver",
+  "com.netflix.spinnaker.orca.kato.pipeline",
```

Comment:
The groovy code also has
```
  "com.netflix.spinnaker.orca.oort.pipeline",
  "com.netflix.spinnaker.orca.oort.tasks",
```
here.

Reply:
It doesn't look like those packages exist anymore. I'm assuming they were removed when they were all refactored into Clouddriver?

Reply:
Good point.